driver.py

```diff
@@ -472,7 +464,7 @@ def generate_probe(p: Panel, options: Options) -> str:
 	s += '''
 	ret = mipi_dsi_attach(dsi);
 	if (ret < 0) {
-		dev_err(dev, "Failed to attach to DSI host: %d\\n", ret);
+		dev_err_probe(dev, ret, "Failed to attach to DSI host\\n");
 		drm_panel_remove(&ctx->panel);
 		return ret;
```

Comment:
According to documentation of dev_err_probe:
```
 * It replaces code sequence::
 *
 * 	if (err != -EPROBE_DEFER)
 * 		dev_err(dev, ...);
 * 	else
 * 		dev_dbg(dev, ...);
 * 	return err;
 *
 * with::
 *
 * 	return dev_err_probe(dev, err, ...);
 ```
 these 3 ines should become 2 lines
 ```
		drm_panel_remove(&ctx->panel);
		return dev_err_probe(dev, ret, "Failed to attach to DSI host\\n");;
 ```

Reply:
Good idea, implemented.